charts/drupal/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: drupal
-version: 1.4.0
+version: 1.4.1
```

Comment:
This is okay, but for future PRs no need to bump chart version as that is part of the release process :)